### ISSUE TITLE
DF version not supported by Dwarf Therapist: Checksums don't match

### ISSUE THREAD

**Issue by sam87** [andrewd18/df-lnp-installer#53]
The version of dwarf fortress as compiled by NFP is not supported by Dwarf therapist, as version checksums don't match:

checksum of compiled version: 0x01645e9a (DF v0.34.11)
checksum Dwar therapist looks for: 0xe066ee88 (DF v0.34.11)

I used df-lnp-installer-0.5.4 to install and compile everything as per readme.

**Comment by andrewd18** [comment]
Samuel,

The DF executable isn't compiled, just downloaded and extracted off of
Toady's site. This is probably an issue on Dwarf Therapist's end. What
version of DT are you running?

**Comment by sam87** [comment]
Currently I'm using DT version 0.6.12 - branch version 20.5, which is downloaded by LNP.

**Comment by sam87** [comment]
Ok, my bad. I had the "Not found: /data/art/mouse.png" error and applied the manual fix from the DF wiki, which involved changing the binaries, which of course then produces a different hash.
The real fix for this problem btw. can be found here: http://mail.bay12games.com/dwarves/mantisbt/view.php?id=2688#c24614
Maybe it is worth to include this? It is not an Issue for Debian 7 but for Fedora 20 (which I am currently using) this is needed to be able to use any tile sets.

**Comment by myk002** [comment]
yeah, I had to remove the prebuild .so files and build libgraphics.so myself on Gentoo.  it does seem like this could be automated so it would work on any modern system
